sr_robot_launch/launch/sr_ur_arms_hand.launch

```diff
@@ -149,7 +150,7 @@
       <arg name="robot_description" value="$(arg robot_description)"/>
       <arg name="pwm_control" value="$(arg pwm_control)"/>
       <arg name="hand_serial" value="$(arg hand_serial)"/>
-      <arg name="hand_id" value="$(arg hand_id)"/>
+      <arg name="hand_prefix" value="$(arg hand_prefix)"/>
```

Comment:
I wonder if it is always better to just send side and have the other prefixes based on that?